### PR TITLE
RQM DLL Updated for Ignore SSL Certificate Code and Added Exception f…

### PR DESCRIPTION
**Pull request by prashelke**
…or Possible Places

Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is pointing to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [x] Code Summary\Comments are added to my code which explains what my code is doing
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for New and Existing Functionality
- [ ] Verify that changes are compatible with all relevant browsers and platforms.
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments
- [ ] Help Libray document is updated accordingly for Feature changes.

### CODE REVIEW

File: Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs [hunk 1]
```diff
@@ -144,7 +144,7 @@ private void UnmapAllRows(object sender, RoutedEventArgs e)
             }
             catch (Exception ex)
             {
-                Reporter.ToLog(eLogLevel.DEBUG, ex.Message);
+                Reporter.ToLog(eLogLevel.DEBUG, ex.ToString());
```
Comment (Maheshkale447) [inline Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs:147]: change it to Reporter.ToLog(eLogLevel.DEBUG, "<Error related msg>", ex);

File: Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs [hunk 2]
```diff
@@ -328,7 +328,7 @@ private void UnMapSelectedElementRowsHandler(object sender, RoutedEventArgs e)
...
```
Comment (Maheshkale447) [inline Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs:331]: change it to Reporter.ToLog(eLogLevel.DEBUG, "", ex);

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs (hunk 1 above)
Comment (prashelke) [reply Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs:147]: Done

File: Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs (hunk 2 above)
Comment (prashelke) [reply Ginger/Ginger/ALM/MapToALMWizard/TestStepMappingPage.xaml.cs:331]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**